changelog/undistributed/ShowInterfacesLink_20210618185223.rst

```diff
@@ -0,0 +1,7 @@
+--------------------------------------------------------------------------------
+                                New
+--------------------------------------------------------------------------------
+* IOSXE
+    * Added ShowInterfacesLink:
+      * show interfaces link
+      * show interfaces {interface} link
```

Comment:
@AlanAutomated thank you very much for contributing to pyATS.
Your parser looks very good. My only recommendation would be to add another unit test to show that your code works with show interfaces {interface} link as well.

Reply:
I added the recommended coverage in 41682c5. The new and existing tests are passing.

![image](https://user-images.githubusercontent.com/84720865/122829843-80fe6000-d2b5-11eb-8c3d-40ab003dbb8c.png)